runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -272,45 +333,43 @@ enddef
 
 # Use when debugger didn't start or ended.
 def CloseBuffers()
-  exe $'bwipe! {ptybuf}'
-  exe $'bwipe! {commbuf}'
-  if asmbuf > 0 && bufexists(asmbuf)
-    exe $'bwipe! {asmbuf}'
+  exe $'bwipe! {ptybufnr}'
+  exe $'bwipe! {commbufnr}'
+  if asmbufnr > 0 && bufexists(asmbufnr)
+    exe $'bwipe! {asmbufnr}'
   endif
-  if varbuf > 0 && bufexists(varbuf)
-    exe $'bwipe! {varbuf}'
+  if varbufnr > 0 && bufexists(varbufnr)
+    exe $'bwipe! {varbufnr}'
   endif
   running = 0
   gdbwin = 0
 enddef
 
 # IsGdbRunning(): bool may be a better name?
-def CheckGdbRunning(): string
-  var gdbproc = term_getjob(gdbbuf)
-  var gdbproc_status = 'unknown'
-  if type(gdbproc) == v:t_job
-    gdbproc_status = job_status(gdbproc)
-  endif
-  if gdbproc == v:null || gdbproc_status !=# 'run'
-    var cmd_name = string(GetCommand()[0])
+def IsGdbStarted(): bool
+  # CHECKME: check this implementation
+  var gdbproc_status = job_status(term_getjob(gdbbufnr))
+  var cmd_name = string(GetCommand()[0])
+  if gdbproc_status !=# 'run'
     Echoerr($'{cmd_name} exited unexpectedly')
```

Comment:
Seems no reason to compute `cmd_name` outside of the `if` and might as well have it directly as interpolated string
```vim
Echoerr($'{string(GetCommand()[0])} exited unexpectedly')
```

Reply:
It's just for a matter of readability.

Reply:
Mostly the comment was about putting
```
var cmd_name = string(GetCommand()[0])
```
inside the `if !=# 'run'`. There's no reason to run the code unless there's a problem. Putting the expr in string interpolate is just a matter of style. (and saving a few cycles, but I tend to way over optimize)

Reply:
I hear you :) 
But in my opinion you lose readability: one thing is to read that you exited unexpectedly because of `cmd_name`, one is to read that you exited because of `string(GetCommand()[0])`. If we all agree, I would keep it as is. There is more room for optimization elsewhere. :)

Reply:
My suggestion is putting the calculation of `cmd_name` inside the if
```vim
  if gdbproc_status !=# 'run'
    var cmd_name = string(GetCommand()[0])
    Echoerr($'{cmd_name} exited unexpectedly')
```
I don't see how calculating a value that is almost never used improves things. If there is an issue with clarity, a comment should suffice.
(this is **not about** putting the expr in string interpolate, that's primarily a style issue)

Reply:
I'll adjust that when merging. Thanks!